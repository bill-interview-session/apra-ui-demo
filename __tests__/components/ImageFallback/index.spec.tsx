import { render } from '@testing-library/react';

import ImageFallback from '@components/ImagePreview/fallback';

const tempImg = `/assets/img/img-loading.svg`

describe('ImageFallback component', () => {
  it('should render snapshot', () => {
    const { container } = render(
      <ImageFallback src={tempImg} fallback={tempImg} height={30} width={30} />,
    );
    expect(container).toMatchSnapshot();
  });
});

