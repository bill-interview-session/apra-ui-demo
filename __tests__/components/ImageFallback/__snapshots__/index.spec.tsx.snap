// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageFallback component should render snapshot 1`] = `
<div>
  <img
    alt=""
    class="sc-eadc27a6-0 cHljDx"
    data-nimg="1"
    decoding="async"
    height="30"
    loading="lazy"
    src="/_next/image?url=%2Fassets%2Fimg%2Fplaceholder.jpg&w=64&q=75"
    srcset="/_next/image?url=%2Fassets%2Fimg%2Fplaceholder.jpg&w=32&q=75 1x, /_next/image?url=%2Fassets%2Fimg%2Fplaceholder.jpg&w=64&q=75 2x"
    style="color: transparent; background-size: cover; background-position: 50% 50%; background-repeat: no-repeat;"
    width="30"
  />
</div>
`;
